IBAnimatableApp/RefreshControlTableViewController.swift

```diff
@@ -36,21 +35,24 @@ public class RefreshTableViewController: AnimatableTableViewController {
     guard time >= 0 else {
       return
     }
-    var attributes: [String : Any] = [:]
-    if let color = self.refreshControlTintColor {
+
+    var attributes = [String: Any]()
+    if let color = refreshControlTintColor {
       attributes[NSForegroundColorAttributeName] = color
     }
     refreshControl.attributedTitle = NSAttributedString(string: "\(Int(time))", attributes: attributes )
 
-    DispatchQueue.main.after(1) { [unowned self] in
-      self.updateMessage(refreshControl: refreshControl, time: time - 1)
+    DispatchQueue.main.after(1) { [weak self] in
```

Comment:
I don't think we need to use `[weak self]` or `[unowned self]` for `DispatchQueue.main`. They are static methods.